multiversx_sdk/converters/transactions_converter.py

```diff
@@ -137,3 +141,16 @@ def _bytes_from_hex(self, value: str) -> bytes:
         if len(value):
             return bytes.fromhex(value)
         return b""
+
+    def _transactions_to_string(self, value: Sequence[ITransaction]) -> Sequence[Dict[str, Any]]:
```

Comment:
this won't be needed anymore if the changes mentioned in the above comment are done

Reply:
removed.